bin/express-cli.js

```diff
@@ -306,12 +325,42 @@ db.sequelize.sync({ force: true }).then(function() {
           app.locals.db = `
 const mongoUri = process.env.MONGODB_URI || 'mongodb://localhost/mydb';
 const mongooseConfigs = { useNewUrlParser: true };
-mongoose.connect(mongoUri, mongooseConfigs)
+mongoose.connect(mongoUri, mongooseConfigs);
 `
           mkdir(dir, 'server/models')
           copyTemplateMulti('js/models/mongoose', dir + '/server/models', '*.js')
       }
 
+      // Caching
+      app.locals.cache = false
+      env.locals.cache = false
+      switch(program.cache) {
+        case 'redis':
+          pkg.dependencies['redis'] = '^3.0.2'
+          app.locals.modules.redis = 'redis'
+          app.locals.cache = `
+/**
```

Comment:
we should look into putting these longer string templates in separate files

Reply:
haha. i was gonna propose that too